css/portal-dashboard/progress-view-legend.less

```diff
@@ -79,3 +78,24 @@
     }
   }
 }
+@media screen and (max-width: 1000px) {
+  .legendContainer {
+    line-height: 25px;
+    bottom: -30px;
+    justify-content: center;
+    flex-wrap: wrap;
+    width: 60%;
```

Comment:
Do we really mean to use 60% when other pieces are static?  Seems like it shouldn't be a percentage.

Reply:
Seems to work though.